runtime/vm/jvminitcommon.c

```diff
@@ -128,22 +128,29 @@ createLoadInfo(J9PortLibrary *portLibrary, J9Pool *aPool, const char *name, U_32
 
 /**
  * This method indicates if the requested jniVersion is valid or not
+ *
+ * Note: a JVMTI environment can be obtained through the JNI Invocation API GetEnv function
+ *       https://docs.oracle.com/en/java/javase/11/docs/specs/jvmti.html
  * @param jniVersion the version to be checked
  * @returns true if the version is valid
  */
 UDATA
```

Comment:
There's no need to change this function at all - it has nothing to do with JVMTI.

Reply:
Moved `JVMTI` version checking out of `jniVersionIsValid()`.